File: Source Code/cpui/lib/beans/responses/create_user_response.dart
class CreateUserResponse {

  String emailId;
  String name;
  String registrationPlate;

  CreateUserResponse({this.emailId, this.name, this.registrationPlate});

  CreateUserResponse.fromJson(Map<String, dynamic> json) {
    emailId = json['emailId'];
    name = json['name'];
    registrationPlate = json['registrationPlate'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['emailId'] = this.emailId;
    data['name'] = this.name;
    data['registrationPlate'] = this.registrationPlate;
    return data;
  }
}

File: Source Code/src/main/java/edu/nwmissouri/mobile/carparking/endpoints/users/responses/CreateUserResponse.java
package edu.nwmissouri.mobile.carparking.endpoints.users.responses;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CreateUserResponse {

    private String emailId;

    private String name;

    private String registrationPlate;
}
